docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+ * 
+ * @property {boolean} dontSetWindowFdc3 For backwards compatibility, `getAgent` 
+ * will set a reference to the Desktop Agent implementation at `window.fdc3` 
+ * if one does not already exist, and will fire the `fdc3Ready` event. Setting  
+ * this flag to `true` will inhibit that behavior, leaving `window.fdc3` unset. 
+ *  
+ * @property {function} failover An optional function that provides a  
+ * means of connecting to or starting a Desktop Agent, which will be called 
+ * if no Desktop Agent is detected. Must return either a Desktop Agent  
+ * implementation directly (e.g. by using a proprietary adaptor) or a  
+ * WindowProxy (e.g a reference to another window returned by `window.open`  
+ * or an iframe's `contentWindow`) for a window or frame in which it has loaded 
+ * a Desktop Agent or suitable proxy to one that works with FDC3 Web Connection 
+ * and Desktop Agent Communication Protocols. 
+ */ 
+type GetAgentParams = { 
+    timeout?: number, 
+    identityUrl?: string, 
+    channelSelector?: boolean, 
+    intentResolver?: boolean,
+    dontSetWindowFdc3?: boolean,
+    failover?: (args: GetAgentParams) => Promise<WindowProxy | DesktopAgent> 
+};
+```
+
+:::note
+
+As web applications can navigate to or be navigated by users to different URLs and become different applications, validation of apps identity is often necessary. The web application's current URL is passed to web browser-based Desktop Agents to allow them to establish the app's identity - usually connecting it with an App Directory record already known to the Desktop Agent. For more details on identity validation see the identity validation section of the  [Web Connection Protocol (WCP)](specs/webConnectionProtocol).
+
+:::
+
+Finally, if there is still no Desktop Agent available, or an issue prevent connection to it, the `getAgent()` function will reject its promise with a message from the [`AgentError`](./Errors#agenterror) enumeration.
+
+## Failover function
+
+Desktop Agent retrieval can time out, for instance if the DA doesn't exist or is unresponsive. The default timeout of 750 milliseconds can be overridden by setting the `timeout` field. An application may also provide a failover function which will be called if an interface cannot be retrieved or times out.
+
+Example: Decreasing the timeout and providing a failover function
+
+```js
+    const desktopAgent = await getAgent({
+        appId: "myApp@yourorg.org",
+        timeout: 250,
+        failover: async (params: GetAgentParams) => {
+            // return WindowProxy | DesktopAgent
+        }
+    }); 
+```
+
+The failover function allows an application to provide a backup mechanism for connecting to a DA. It is called only when establishment through normal procedures fails or times out.
+
+:::note
+
+If you wish to _completely override FDC3s standard mechanisms_, then do not use a failover function. Instead, simply skip the `getAgent()` call and provide your own DesktopAgent object.
```

Comment:
- Change "FDC3s" to "FDC3's"
- Also, this might be clearer if it was something along the lines of "FDC3's standard discovery/acquisition mechanisms". Or use the word "installation" if you don't like that.  The reason being that we're not talking about override everything about FDC3 - just the discovery/acquisition/installation step.

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85